views/map/index.android.js

```diff
@@ -29,8 +29,7 @@ export default class OlafMapView extends React.Component {
           <Button
             onPress={() => Linking.openURL(mapInfo.url).catch(err => console.error('An error occurred', err))}
             style={styles.button}
-          >
-          See the map!</Button>
+          > View Map </Button>
```

Comment:
I'll bet it's the `width` of the button – is it collapsing down to a width smaller than the normal length of the string? If so, it might be wrapping to two lines, and Button could be cutting it off.

Alternately, what if you wrap the text in a `<Text>` node?

`><Text>View Map</Text></Button>`

If that doesn't work, i'll merge it and stab it later.